src/api/cityConfigs.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import type {$Request, $Response} from 'express'
+import express, {Router} from 'express'
+import path from 'path'
+import HttpStatus from 'http-status-codes'
+import cityConfigs from '../cities/cityConfigs'
+
+export default (): Router => {
+  const router = new Router()
+
+  router.get('/', (request: $Request, response: $Response) => {
+    response.status(HttpStatus.OK).json(cityConfigs)
+  })
+
```

Comment:
I think, another route would be interesting: `/:hostName` to deliver only the data, that is needed for the actual city we're looking for...